aws-scala-workshop/src/main/scala/com/dellemc/ecs/s3/sample/_00_CreateBuckets.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2018 Dell Inc. or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package com.dellemc.ecs.s3.sample
+
+import com.amazonaws.services.s3.AmazonS3
+
+object _00_CreateBuckets extends BucketAndObjectValidator {
+
+    def main(args: Array[String]): Unit = {
+        createBucket(AWSS3Factory.getS3ClientWithV2Signatures(), AWSS3Factory.S3_BUCKET)
+        createBucket(AWSS3Factory.getS3ClientWithV4Signatures(), AWSS3Factory.S3_BUCKET_2)
```

Comment:
We do need to show how to use v2 signatures, but why are we creating 2 buckets?  I think we should parameterize this or comment one of these lines out.  Doing all of these examples twice is confusing.

Reply:
revised